Add depth control preset option

mdhorn opened the pull request:
Changes proposed in this pull request:
- add depth control preset to dynamic reconfigure options for r200
- add individual depth control options to dynamic reconfigure options for r200
- glue code to update individual options when preset select; also invalidate preset when individual option is changed

anelliot commented:
Looks good to me. √

reaganlo commented on realsense_camera/src/r200_nodelet.cpp:
```diff
@@ -114,11 +116,150 @@ namespace realsense_camera
+   */
+  void R200Nodelet::setDynamicReconfigDepthControlPreset(int preset)
+  {
+    // There is not C++ API for dynamic reconfig so we need to use a system call
```
Typo - "no" instead of "not" :)

reaganlo commented on realsense_camera/src/r200_nodelet.cpp:
```diff
@@ -114,11 +116,150 @@ namespace realsense_camera
+    std::string current_dc;
+    std::string option_value;
+
+    // There is not C++ API for dynamic reconfig so we need to use a system call
```
Typo - "no" instead of "not" :)

reaganlo reviewed (approved):
Phew.. Thats some code that you have written.. :)

anelliot committed:
Add depth control preset option

Added individual Depth Control options and preset as
a group in dynamic reconfigure.

Modified unit tests to check for new default settings.
